petropipe/crud_events/job_card_master.py

```
@@ -0,0 +1,25 @@
+            "company": doc.company or ''
+        }, update_modified=True)
+    else:
+        try:
+            job_card_master = frappe.new_doc("Job Card Master")
+            job_card_master.sales_order = doc.name
+            job_card_master.job_no = doc.job_no
+            job_card_master.customer = doc.customer or ''
+            job_card_master.incoterm = doc.incoterm or ''
+            job_card_master.company = doc.company or ''
+            job_card_master.insert(ignore_permissions=True)
+        except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)